crossandeancommands/make_nexus.py

```diff
@@ -0,0 +1,82 @@
+"""
+Create a nexus file from the most recent data on Edictor.
+"""
+from pyedictor import fetch
+from lingpy import Wordlist
+from lingpy.convert.strings import write_nexus
+
+def run(args):
+
+    with open("./outputs/crossandean.tsv", "w",
+            encoding="utf-8") as f:
+            f.write(fetch("crossandean",
+                    columns=[
+                            "ID", 
+                            "ALIGNMENT", 
+                            "COGIDS", 
+                            "CONCEPT", 
+                            "DOCULECT", 
+                            "FORM", 
+                            "TOKENS", 
+                            "BORROWING", 
+                            "SOURCE", 
+                            "SUBGROUP"
+                            ]
+                    ))
+
+    wl = Wordlist('./outputs/crossandean.tsv')                        
+    
+
+    # These are the blacklists for the nexus file
+    blacklist_concepts = [
```

Comment:
The black list is because they re-iterate via same cogid? Can we automate this?

Reply:
To get the subgroup, you need to specify "language_subgroup" in the columns (as this is the internal representation). To make "subgroup" appear in the wordlist, add `("language_subgroup", "subgroup")` to the namespace keyword.

Reply:
This blacklist has concept-specific reasons. For some, we have reasonable doubt that there are problems with the original data (e.g. mixing up `green` as "unripe fruit" and "color"). For others we have a high amount of colexification (e.g. star: lucero/estrella, or arm/branch). As the reasons are specific for each concept, I don't think we can get a good automation for this. I would prefer to discuss this explicitly in our article instead.

Reply:
What could be done is to filter for subgroups as an argument for the command, or provide a boolean argument for all rows that have any entry in borrowings. But this would not be used right now.